src/python/pants/backend/go/lint/gofmt/subsystem.py

```diff
@@ -11,3 +11,4 @@ class GofmtSubsystem(Subsystem):
     help = "Gofmt-specific options."
 
     skip = SkipOption("fmt", "lint")
+    args = ArgsListOption(example="-s")
```

Comment:
Can we incorporate the list of supported args in this help-string?

Reply:
Excellent idea, thank you! This is now done :)